packages/preview/peace-of-posters/0.4.3/typst.toml

```diff
@@ -9,3 +9,7 @@ repository = "https://github.com/jonaspleyer/peace-of-posters"
 homepage = "https://jonaspleyer.github.io/peace-of-posters/"
 categories = ["poster"]
 
+[template]
+path = "templates/pop-uni-fr-poster"
+entrypoint = "main.typ"
+thumbnail = "templates/pop-uni-fr-poster/peace-dove-thumbnail.png"
```

Comment:
The thumbnail file should not be in the template folder, so that it is not copied to the projects created from the template.

Reply:
The thumbnail is coincidentally also used by the template itself. Do you want me to copy the file to another location anyway?